### DISCUSSION TITLE
Is there a way to ignore tables?

### DISCUSSION THREAD

**Discussion by alexdor** [stephenafamo/bob#274] [Q&A] [answered]
Hi everyone,

I've been looking through the config for code generation of bob, and I can't seem to find a way to not generate bindings for a table. Is there a way that I'm not seeing?

My use case is that I want bob to not generate models for a migrations table.

**Comment by stephenafamo** [comment, marked as answer]
Yes, there is the `only` and `except` keys under each driver.

https://bob.stephenafamo.com/docs/code-generation/psql
